doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc

```diff
@@ -58,4 +58,12 @@ kieservers:
 ----
 $ java -jar {PRODUCT_FILE}-process-migration-service-standalone.jar -s./config.yml
 ----
+. To enable auto-detection of a JDBC driver by Thorntail, add the `JAR` file of the JDBC driver name to the `thorntail.classpath` system property. For example:
++
+[source,subs="attributes+"]
+----
+$ java -Dthorntail.classpath=./h2-{PRODUCT_FILE}-process-migration-service-standalone.jar -s./config.yml
```

Comment:
The name of the jdbc driver is a little misleading, this will end up in a name like `h2-rhpam-7.11.0-process-migration-service-standalone.jar`. In this example, we should provide a proper H2 JDBC driver name, like `h2-1.4.200.jar` or any other different JDBC drive like `mariadb-java-client-2.4.2.jar`